### DISCUSSION TITLE
[Showcase] Data Marketing School

### DISCUSSION THREAD

**Discussion by lucasgtm** [gethyas/hyas#456] [Showcase]
### Site name

Data Marketing School

### Public URL

https://data-marketing-school.com

### Short description

Uncover the secrets your website hides from you.
Collect, analyze and visualize your visitors' acquisition, behavior and conversion data while respecting their privacy.

### Terms

- [X] I agree to the [Terms of Service](https://gethyas.com/terms/).

**Comment by h-enk** [comment]
Thanks for sharing — nice!
